src/backends/torch/native/native_factory.cc

```diff
@@ -48,8 +48,9 @@ namespace dd
   {
     if (tdef.find("nbeats") != std::string::npos)
       {
-        std::vector<std::string> p;
         double bc_loss_coef = NBEATS_DEFAULT_BACKCAST_LOSS_COEFF;
+        std::vector<std::string> p{ "t2", "s", "g3", "b3", "h10" };
```

Comment:
Can you add a comment on what this means, FTR ?

Reply:
ok, done